scripts/rv_expression_association/get_vep_variants.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import logging
+import hail as hl
+import numpy as np
+import pandas as pd
+from hail.methods import export_plink
+from cpg_utils.hail_batch import dataset_path, init_batch, output_path
+
+# object containing variants within a 50K window on either side of the VPREB3 gene
+MT = dataset_path('v0/VPREB3_50K_window.mt')
+VEP_HT = dataset_path('v0/VPREB3_50K_window_vep.ht')
+
+# use logging to print statements, display at info level
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+
+
+def main():
+    # read and densify object
+    init_batch()
+    mt = hl.read_matrix_table(MT)
+    mt = hl.experimental.densify(mt)
+
+    logging.info('Number of variants in window: {}'.format(mt.count()[0]))
```

Comment:
`mt.count()` might lead to the expensive `densify` operation being run redundantly here. Consider running `densify` _after_ this print statement instead, to avoid that duplication, as it won't change the row count.